doc/mediasdk-man.md

```diff
@@ -4139,16 +4139,16 @@ The application can attach this extended buffer to the [mfxVideoParam](#mfxVideo
 `GPB` | Turn this option OFF to make HEVC encoder use regular P-frames instead of GPB.<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option
 `LowDelayHrd` | Corresponds to AVC syntax element low_delay_hrd_flag (VUI).<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option.
 `MotionVectorsOverPicBoundaries` | When set to OFF, no sample outside the picture boundaries and no sample at a fractional sample position for which the sample value is derived using one or more samples outside the picture boundaries is used for inter prediction of any sample.<br><br>When set to ON, one or more samples outside picture boundaries may be used in inter prediction.<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option.
-`MaxFrameSizeI` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only I-frames.
-`MaxFrameSizeP` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only P-frames.
+`MaxFrameSizeI` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only I-frames. Maximum allowed frame size per profile/level will be calculated in driver and be used when MaxFrameSizeI and MaxFrameSizeP are both set to 0. Feature, when MaxFrameSizeI is not specified and MaxFrameSizeP is specified, is unsupported.
```

Comment:
IMO, "in driver" is bad wording. MediaSDK manual is hw agnostic.

Reply:
Please re-formulate this "Feature, when MaxFrameSizeI is not specified and MaxFrameSizeP is specified, is unsupported.". Does it mean that MaxFrameSizeI must be set if MaxFrameSizeI is set?

Reply:
@stasefremov It means that MaxFrameSizeI must be set if MaxFrameSizeP is set

Reply:
Then I think description "MaxFrameSizeI must be set if MaxFrameSizeP is set" is much more clear than "Feature, when MaxFrameSizeI is not specified and MaxFrameSizeP is specified, is unsupported." :)

Reply:
"Maximum allowed frame size per profile/level will be calculated in driver and be used when MaxFrameSizeI and MaxFrameSizeP are both set to 0.". I think AVC spec limitations will still be applied even when respective MaxFrameSizeX is set. E.g. if MaxFrameSizeI/MaxFrameSizeP is set to value bigger than spec limitation. Also spec limitation will be applied to P-frames if MaxFrameSizeI is set and MaxFrameSizeP isn't set. So current statement looks too narrow.

Reply:
@stasefremov Do you mean, we need to mention, if both values ​​are not set or exceed the max value, then the MaxFrameSizeX will be default?

Reply:
"if both values ​​are not set or exceed the max value, then the MaxFrameSizeX will be default?" It's much closer but still not there. There is still word "both" which looks incorrect to me. I don't think we apply AVC spec limitation for frame size ONLY if BOTH MaxFrameSizeI/MaxFrameSizeP are not set.  If MaxFrameSizeI is set and MaxFrameSizeP is not set, we still apply AVC spec limitation for P-frames, right?